docs/configuration/translation-strings.md

```diff
@@ -23,29 +11,26 @@ You can also translate [Collection Content]() and [Field Names]().
 </video>
 
 When configuring Fields as well as Presets and Bookmarks, you may wish to add additional notes or descriptions to
-clarify their purpose or context.
+clarify their purpose or context. To create and assign a Translation String, follow these steps.
 
-**Create and Manage Translation Strings**\
+**Create and Manage Translation Strings**
 
 1. Navigate to **Settings > Translation Strings**.
 2. Click <span mi btn>add</span> in the page header and a side menu will appear.
-3. Add a key and click **"Create New"**. Another side menu will appear.
+3. Add a key and click **"Create New"** and another side menu will appear.
 4. Select the language and add the corresponding translation.
 5. Click <span mi btn>check</span> to confirm and add the translation.
 6. Repeat steps 3-5 as desired.
 7. Optional:
    - Click and drag <span mi icon>drag_handle</span> to reposition a translation.
```

Comment:
I'll let @rijkvanzanten be the tie-breaker here 😄

in case it is kept, the general idea is to provided the flexibility of re-ordering them based on user preference such as making their preferred/frequent-tweaked language on top. But understandably this can be rather niche, unless they are translating dozens of languages to begin with.

Reply:
This needs to go out asap and it's impossible to remove it from a video...
I will cut the text referring to the feature, then replace the video in a few weeks, sans *"grabby bars"*.

Reply:
@erondpowell the sorting has been removed in #13920 👍 you can remove any mentions of it safely now.